packages/gnome-shell/src/extensions/sharedInternals.d.ts

```diff
@@ -177,3 +173,15 @@ declare class Console {
 
     groupEnd(): void;
 }
+
+declare interface Console extends ConsoleLike {}
+
+/**
+ * @see https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/extensions/sharedInternals.js#L285
+ * @version 48
+ */
+declare class Console implements ConsoleLike {
```

Comment:
Let's just remove this? It's of no use to anyone, since it's not exported in GNOME Shell 🤷

We only need the interface.